packages/extension-ui/src/util/languages.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { TFunction } from 'i18next';
+
+interface Option {
+  info?: string;
+  isDisabled?: boolean;
+  isHeader?: boolean;
+  text: React.ReactNode;
+  value: string | number;
+}
+
+export default function create (t: TFunction): Option[] {
+  return [
+    {
+      text: t<string>('lng.detect', 'Default browser language (auto-detect)', { ns: 'apps-config' }),
```

Comment:
```suggestion
      text: t<string>('lng.detect', 'Browser language (auto-detect)'),
```

Reply:
Probably (if going for auto-detection) it would make sense to just remove the "(auto-detect)" rather.

Reply:
Sounds good, I removed the language detection.